internal/backends/python/gen_pypi_map/gen_pypi_map.go

```diff
@@ -83,11 +85,40 @@ func cmd_test(args []string) {
 		}
 		bqCache = normalizedBqCache
 
-		packageList := []string{}
+		packageRemaps := make(map[string]string)
+		file, err := os.Open(*testRemapFile)
+
+		if err == nil {
+			scanner := bufio.NewScanner(file)
+			for scanner.Scan() {
+				columns := strings.SplitN(scanner.Text(), ",", 3) // old,new,description
+				if len(columns) > 0 {
```

Comment:
```suggestion
                                old := columns[0]
                                new := columns[1]
				if len(columns) > 0 {
```
You might consider doing something like this to avoid the comment and keeping track of indexes but very minor.